nemo/collections/nlp/modules/common/megatron/utils.py

```diff
@@ -417,8 +417,8 @@ def split_list(inputs, num_chunks):
     """
     Split a list into equal sized chunks
     """
+    # if len(inputs) % chunk_size != 0, round down the chunk size
     chunk_size = len(inputs) // num_chunks
-    assert len(inputs) % chunk_size == 0, "Issue with batch size configuration!"
```

Comment:
need to ask someone from NLP to make sure they're fine with removing these asserts

Reply:
acknowledge. will do